app/util/confluence/populate_db.sh

```diff
@@ -20,7 +20,7 @@ CONFLUENCE_DB_PASS="Password1!"
 SELECT_CONFLUENCE_SETTING_SQL="select BANDANAVALUE from BANDANA where BANDANACONTEXT = '_GLOBAL' and BANDANAKEY = 'atlassian.confluence.settings';"
 
 # Confluence version variables
-SUPPORTED_CONFLUENCE_VERSIONS=(6.13.13 7.0.5 7.4.5)
+SUPPORTED_CONFLUENCE_VERSIONS=(7.4.5 7.0.5 6.13.13)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.